src/other/Table.js

```diff
@@ -134,7 +136,13 @@
     Table.prototype.update = function (domNode, element) {
         HTMLWidget.prototype.update.apply(this, arguments);
         var context = this;
-       
+        if(this.fixedSize()){
+            var box = d3.select(".tableDiv > table").node().getBoundingClientRect();
+            element.style({width: box.width + "px",height: box.height + "px"});
```

Comment:
I don't follow what you're suggesting... removing the width + height styles of this.element()?

The goal of lines 140 and 141 are to grab the width and height of the <table> and apply them to the target's <div>.